README.md

```diff
@@ -24,3 +30,36 @@ Running during normal development:
 
 While the service is running, you can see some basic back end swagger API documentation at 
 * http://localhost:8080/v2/api-docs
+
+#### Front End
+
+What you need:
+
+* Nodejs (only if you don't use Vagrant) - https://nodejs.org/en/download/
+* (Optional) Vagrant - https://www.vagrantup.com/downloads.html
+* (Optional) VirtualBox - https://www.virtualbox.org/wiki/Downloads
+
+######  If you have chosen to install both Vagrant and VirtualBox
+Run the following commands in a terminal to get started:
+```sh
+
+$ cd <code dir>/team9sofine
+$ npm install
+$ gulp build
+```
+
+Now, in your browser, go to `http://192.168.33.110` and you should see the frontend loaded. Within the terminal, you can also run `$lynx myapp.dev` to see a text-only version. Ctrl+C to quit out.
```

Comment:
Should this be 'localhost' or 127.0.0.1 or ::1 ? If not, I'm missing something unless this is one of those magic addresses